### ISSUE TITLE
Polish translation

### ISSUE THREAD

**Issue by pierd** [google/comprehensive-rust#399]
Polish translation of the Rust Book for reference: https://github.com/beling/book-pl

**Comment by mgeisler** [comment]
Hi @pierd, thanks a lot for working on this! I've added it to the list of ongoing translations in #282. Please let me know if you run into any trouble with the infrastructure.

When you have a repository setup, you could post it here so that other interested people can easily find it and help out.

**Comment by pierd** [comment]
Hi @mgeisler, I've just started with some automatic translations using cloud-translate and checked a few to see some results when using `mdbook`. It's currently in this branch: https://github.com/pierd/comprehensive-rust/tree/pl

BTW one thing that confused me initially was that I didn't see any translations after running cloud-translate even though I was pointing `mdbook serve` to the correct `.po` file, it doesn't use translations that are marked as `fuzzy` (in .po) or "Needs work" (in Poedit). Once I verified some in Poedit I've started seeing them in the render website.

**Comment by mgeisler** [comment]
Hi @pierd, awesome, thanks for linking the repository!

> BTW one thing that confused me initially was that I didn't see any translations after running cloud-translate even though I was pointing `mdbook serve` to the correct `.po` file, it doesn't use translations that are marked as `fuzzy` (in .po) or "Needs work" (in Poedit). Once I verified some in Poedit I've started seeing them in the render website.

Oh yeah, I can see how that would be very confusing! Can I get you to add a sentence about the `fuzzy` marker to `TRANSLATIONS.md`? I see that @fechu also asked about the same thing for the German translation just yesterday :smile:

**Comment by pierd** [comment]
@mgeisler added here: https://github.com/google/comprehensive-rust/pull/466 (also mentioned `cloud-translate`)

**Comment by pierd** [comment]
Hi @mgeisler, I had some problems with machine translation in the end - it translated only part of the text. I did go through a bunch of translations by hand. Let me try the machine one again. In any case I'll open a PR with what I have so far.

**Comment by mgeisler** [comment]
> I had some problems with machine translation in the end - it translated only part of the text.

Okay, feel free to open an issue about this in https://github.com/mgeisler/cloud-translate if you have more details.

In any case, my impression is that the machine translations need a lot of work afterwards.

**Comment by pierd** [comment]
It worked well this time. I think I just run it once the first time.

**Comment by mgeisler** [comment]
Ah, good! So you had to run it multiple times to get everything translated?

I did it like this to give people a chance to control the cost (you're charged per character) and because the API limits the amount of data you can send at once (I could of course have dealt with that internally, but it was simpler not to).

**Comment by pierd** [comment]
Yes, in the end I had to run it a couple more times until it said that 0 messages were translated.
